dists/android/build.gradle

```diff
@@ -44,7 +44,7 @@ android {
         targetSdkVersion 34
 
         versionName "2.9.0"
-        versionCode 2090001
+        versionCode 2090000
```

Comment:
On a second thought, this may cause troubles to users installing the APK.
We may have to adopt this specific change in the next version.

Reply:
Could you elaborate on why this could cause troubles to the APK users? 
APKs per arch would have the 1, 2, 3, 4, end digits (will we support a fat APK too?).
In any case, if they manually install the APK over the FAT bundle (why do that though?), the upgrade should work (versionCode ending at 0 upgrading to equal (if we have FAT APK) or greater value for versionCode).

Reply:
The change above (which I reverted) set the last digit of the versionCode to 0.
This version is used on every buildbot builds as we don't change the version code for such builds.